### PR TITLE
Delete vm scan test from test_rest and modify vm scan tests in test_v…

### PR DESCRIPTION
**Pull request by lehinevych**
{{pytest: cfme/tests/infrastructure/test_vm_analysis.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 602ea8c4c0c01724d8b98075eeea16c076ec86ba:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_vm_analysis.py [hunk 1]
```diff
@@ -391,16 +391,8 @@ def _scan_test(provider_crud, vm, os, fs_type, soft_assert, rest="using_ui", res
...
     def test_stopped_vm(
-            self,
-            provider,
-            vm,
-            vm_name,
-            verify_vm_stopped,
-            os,
-            fs_type,
-            soft_assert):  # , register_event):
+            self, provider, vm, vm_name, verify_vm_stopped, os, fs_type, soft_assert):
```
Comment (psav) [inline cfme/tests/infrastructure/test_vm_analysis.py]: ~Oh goody!

**Comment by dajoRH** [comment]
Lint report for commit a32f71c7e51f3590bd7ceda4b3bd8b94daa79b05:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by lehinevych** [comment]
Closed in favor to #2732

**Pull request closed without merging**